src/rpc/core_rpc_server.h

```diff
@@ -320,6 +322,9 @@ namespace cryptonote
 private:
     bool check_core_busy();
     bool check_core_ready();
+
+    template<typename Response>
```

Comment:
`template <typename response>`
to stick to conventions of the codebase.

Reply:
updated